README.md

```diff
@@ -4049,5 +4049,9 @@ Does anyone loves archery and choppers the same as I do? :)
 My name is [Kostiantyn](https://github.com/rukhlovka)! And this is my cat and my hands when I was learning Git.
 ![Kitten](./assets/images/cat-Mini.jpg)
 
+
+## Hi! i'm Andre! 
+![Screenshot-image-link](./assets/images/Bandit.jpg) 
```

Comment:
@artem-trubin noticed on time; it seems like the link is broken.
Have you forgotten to add the file?

Reply:
@suchov Hi, I fixed the problem. Please, review.